library/CodeMonkeysRu/GCM/Exception.php

```diff
@@ -9,5 +9,7 @@ class Exception extends \Exception
     const MALFORMED_REQUEST = 3;
     const UNKNOWN_ERROR = 4;
     const MALFORMED_RESPONSE = 5;
-
-}
\ No newline at end of file
+    const INVALID_DATA_KEY = 6;
+    const CURL_ERROR = 7;
```

Comment:
Ideally we should not expose our transport layer.
There should be no mentions of CURL